### DISCUSSION TITLE
Does HelmRepository CRD support AWS ECR helm chart repositories?

### DISCUSSION THREAD

**Discussion by nicogl** [fluxcd/flux2#978] [Q&A] [answered]
I'm trying to use a Helm repository I created in AWS ECR but I'm getting unauthorized errors, even when I set a secret with credentials.
Does HelmRepository CRD actually support Helm repositories deployed in AWS ECR?

**Comment by hiddeco** [comment, marked as answer]
We do at present not support OCI charts, see: https://github.com/fluxcd/source-controller/issues/124
